proposals/feature-dependencies.md

```diff
@@ -116,22 +115,35 @@ Published Features referenced directly by [HTTPS URI pointing to the packaged ta
 
 ### (B) `dependsOn` install order algorithm
 
-The orchestrating tool is responsible for calculating a Featuure installation order (or error, if no valid installation order can be resolved). The set of Features to be installed is the union of user-defined Features and their dependencies.  The orchestrating tool will perform the following steps:
+The orchestrating tool is responsible for calculating a Feature installation order (or error, if no valid installation order can be resolved). The set of Features to be installed is the union of user-defined Features and their dependencies.  The orchestrating tool will perform the following steps:
+
+#### **(B1) Building dependency graph**
+
+From the user-defined Features, the orchestrating tool will build a dependency graph.  The graph will be built by traversing the `dependsOn` and `installsAfter` properties of each Feature.  The metadata for each dependency is then fetched and the node added as an edge to to the dependent Feature.  For `dependsOn` dependencies, the dependency will be fed back into the worklist to be recursively resolved. 
+
+An accumulator is maintained with each new Feature that has been discovered, as well as a pointer to its dependencies.  If the exact Feature (see **Feature Equality**) has already been added to the accumulator, it will not be added again.  The accumulator will be fed into (B3) after all the Feature tree has been resolved.
+
+The graph may be stored as an adjacency list with two kinds of edges (1) `dependsOn` edges or "hard dependencies" and (2) `installsAfter` edges or "soft dependencies".
 
-#### (B1) Building dependency graph
+#### **(B2) Assigning `roundPriority`**
 
-From the user-defined Features, the orchestrating tool will build a dependency graph.  The graph will be built by traversing the `dependsOn` property of each Feature and recursively resolving each Feature from the property.  An accumulator is maintained with each new Feature that has been discovered, as well as a pointer to its dependencies.  If the exact Feature (see **Feature Equality**) has already been added to the accumulator, it will not be added again.  The accumulator will be fed into (B2) after all the Feature tree has been resolved.
+Each node in the graph will originally be assigned a default `roundPriority` of 0.  
```

Comment:
```suggestion
Each node in the graph will originally be assigned a default `roundPriority` of 0 (**).  
```

Reply:
Suggestion: I believe it's not always true (see :L134)? Something like this might help avoid possible confusion.